### PR TITLE
Create berlin-blockchain-week-crypto-info-brunch.md

### PR DESCRIPTION
**Pull request by laeccles**
@WinsCodes New Event. Can you add this logo for the host logo 
impossiblecloud_logo

### CODE REVIEW

**Review by laeccles — COMMENTED** [review]
updated hosts

**Review by laeccles — COMMENTED** [review]
Description update

**Comment by WinsCodes** [comment]
@laeccles I can see 4 hosts - Impossible Cloud, Gnosis, Hats Finance and HV Capital. Can yu please mention which host does the above shared logo is linked to. Also please provide logos for the other remaining 3 hosts.

File: content/events/berlin-blockchain-week-crypto-info-brunch.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#Preferred means of contacts - this is a placeholder for email(for eg:  - email|mailto:<email_id>) and other social handles like Twitter, LinkedIn, Discord, etc. (for eg.   - 'twitter|https://twitter.com/IPFS/status/1629199396700098560?s=20')
+
+preferredContacts:
+  - '<contact_means>|<contact_url>'
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - Impossible Cloud|<host_logo>
```
Comment (WinsCodes) [inline content/events/berlin-blockchain-week-crypto-info-brunch.md:42]: @laeccles I can see 4 hosts - Impossible Cloud, Gnosis, Hats Finance and HV Capital. Can yu please mention which host does the above-shared logo is linked to? Also please provide logos for the other remaining 3 hosts.  Cc @pandeyanujk

Comment (laeccles) [reply content/events/berlin-blockchain-week-crypto-info-brunch.md:42]: The host previously linked is Impossible Cloud

File: content/events/berlin-blockchain-week-crypto-info-brunch.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+eventType: "Social" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Berlin, Germany"  #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+venueName: "Full Node" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "Glogauer Str. 6, 10999 Berlin, Germany" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "[<Google_Map_link_for_the_venue>](https://maps.app.goo.gl/P1UjhUywPmMxpP3C7)" #the event venue Map link (will show up on a map) or just leave it blank
```
Comment (WinsCodes) [inline content/events/berlin-blockchain-week-crypto-info-brunch.md:22]: @laeccles This map link does not seem to work. Can yu please check this.

**Comment by WinsCodes** [comment]
Event added using the pull request - https://github.com/memser-spaceport/pln-events/pull/476

**Pull request closed without merging**